draft-ietf-mls-architecture.md

```diff
@@ -696,6 +696,109 @@ software is updated while the client is a member of a group. Thus, in addition
 to preventing downgrade attacks, the members of a group can also observe when it
 is safe to upgrade to a new ciphersuite or protocol version.
 
+# Interoperability Requirements
```

Comment:
It seems like these requirements apply not just for cases where two deployments need to interop, but even within a single deployment, in the sense that these are all choices that the deployment has to make.  Maybe something like:

```
# Operational Requirements

MLS is a security layer that needs to be integrated with an application. An
fully-functional deployment of MLS will have to make a number of decisions about
how MLS is configured and operated.  Deployments that wish to interoperate will
need to make compatible decisions. This section lists all of the dependencies of
an MLS deployment that are external to the protocol specification, but would
still need to be aligned within a given MLS deployment, or for two deployments
to potentially interoperate. 
```